feature-ledger-api/src/main/java/io/novafoundation/nova/feature_ledger_api/data/repository/LedgerDerivationPath.kt

```diff
@@ -9,4 +9,8 @@ object LedgerDerivationPath {
     fun derivationPathSecretKey(chainId: ChainId): String {
         return "$LEDGER_DERIVATION_PATH_KEY.$chainId"
     }
+
+    fun genericDerivationPathSecretKey(): String {
+        return "$LEDGER_DERIVATION_PATH_KEY.Generic"
```

Comment:
Decided to store DP for generic app accounts the same way we store it for legacy apps. We will need to adapt cloud backups to support this later